styleguide/components/search/_search.html.haml

```diff
@@ -1 +1 @@
-= render partial: "@citizensadvice/search", locals: {search_action_url: '/search/page' }
+= render partial: "@citizensadvice/design-system/haml/search", locals: { search: { action_url: "/search/page", search_param: "q" } }
```

Comment:
If we allow a consumer of the design system to specify the name of the search param, should we should do the same with the search box in the header?  Otherwise they could potentially have to deal with catering for two names for the same param.

In reality, I think most consumers will just be pointing back to the search page on the public advice site, so is there any advantage in making this configurable?

Reply:
Good point, I agree. I think I'll remove that change from this PR as it's tangential to what I'm trying to fix here anyway.